pages/custom-query-modules/cpp/cpp-api.md

```diff
@@ -2354,6 +2354,54 @@ Additionally, operator<< is overloaded for Type enum, and usage of this operator
 std::ostream &operator<<(std::ostream &os, const mgp::Type &type)
 ```
 
+## Database internals API
+
+This section covers the C++ API methods for interfacing with the database other
```

Comment:
This section describes C++ API methods for database operations beyond graph manipulation.